store/resource.js

```diff
@@ -4,7 +4,7 @@ import apiRequest from 'api'
 
 class Resource {
   static defaultOptions = {
-    request: (...args) => apiRequest(args).then(({ data }) => data),
+    request: (...args) => apiRequest(args).then((data) => data),
```

Comment:
It looks like you don't need a repeating function here.

```suggestion
    request: (...args) => apiRequest(args),
```

And now it looks like you do not need to override `request` at all, do you?

```suggestion
    request: apiRequest,
```

Unless you want to bind the `apiRequest`. However, it does not accept the context.